### ISSUE TITLE
[med] Color Contrast: Many Pages: Grey icons against white background fail contrast requirements. 

### ISSUE THREAD

**Issue by Stutikuls** [Expensify/App#8528]
## Action Performed:
1. Using Chrome, open URL staging.new.expensify.com
2. Navigate to Setting using Tab key and press Enter to select it.
3. Navigate to any workspace using Tab key and press using Enter to select it.
4. Notice More button.
5. Measure the color contrast ratio of the grey icons against white background for more button.

## Expected Result:
The visual presentation of meaningful graphical objects or interactive components should have a contrast ratio of at least 3.00:1 with the surrounding background color.

## Actual Result:
The grey icons (#C6C9CA) on white background (#FFFFFF) have a color contrast ratio of 1.5:1.

## Workaround:
Yes

## Area issue was found in:
Settings -> More

## Failed WCAG checkpoints
1.4.11

## User impact:
The low-vision and color-blind users might not be able to see the icons.

## Suggested resolution:
Use a darker shade of grey for the icons to achieve a color contrast ratio of at least 3.00:1. One example of color to use is #8D9596, which will give a color contrast ratio of 3.05:1.

## Platform:
- Web
- iOS
- Android
- Desktop App
- Mobile Web

**Version Number:**v1.1.49-1
**Reproducible in staging?:** Yes
**Issue reported by:** Stuti

![7956_CCA fail for more control](https://user-images.githubusercontent.com/97600950/162180969-f4dde0fe-7461-4f35-84a0-657a9fd86c74.png)
